rasa/core/processor.py

```diff
@@ -480,9 +480,19 @@ def is_action_limit_reached():
 
     # noinspection PyUnusedLocal
     @staticmethod
-    def should_predict_another_action(action_name, events) -> bool:
-        is_listen_action = action_name == ACTION_LISTEN_NAME
-        return not is_listen_action
+    def should_predict_another_action(action_name: Text, events: List[Event]) -> bool:
+        """Determine whether the processor should predict another action.
+
+        Args:
+            action_name: Name of the latest executed action.
+            events: List of events returned by the latest executed action.
+
+        Returns:
+            `False` if `action_name` is `ACTION_LISTEN_NAME` or
+            `ACTION_SESSION_START_NAME`, otherwise `True`.
+        """
+
+        return action_name not in (ACTION_LISTEN_NAME, ACTION_SESSION_START_NAME)
```

Comment:
optional: alternatively, we could also pass `tracker.last_action ` as `action_name` (then you can keep the condition here as before).